Debug magma on crusher

mewall opened the pull request:
Modifications to support cmake package bug fix for BML on OLCF crusher

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/lanl/qmd-progress/244)
<!-- Reviewable:end -->

nicolasbock commented on build.sh:
```diff
@@ -35,7 +35,6 @@ EOF
     echo "CC                 Path to C compiler       (default is ${CC})"
     echo "CXX                Path to C++ compiler     (default is ${CXX})"
     echo "FC                 Path to Fortran compiler (default is ${FC})"
-    echo "BML_ROCSPARSE      Build with rocSPARSE     (default is ${BML_ROCSPARSE})"
```
Why do you remove this line? The variable `BML_ROCSPARSE` is still being used.

mewall commented:
Thanks for catching that. It's not needed any more with the BML package revision. I changed CMakelists.txt and removed other refs to BML_ROCSPARSE in build.sh.

________________________________
From: Nicolas Bock ***@***.***>
Sent: Monday, February 6, 2023 8:40:46 PM
To: lanl/qmd-progress
Cc: Wall, Michael E; Author
Subject: [EXTERNAL] Re: [lanl/qmd-progress] Debug magma on crusher (PR #244)


@nicolasbock requested changes on this pull request.

________________________________

In build.sh<https://urldefense.com/v3/__https://github.com/lanl/qmd-progress/pull/244*discussion_r1098141880__;Iw!!Bt8fGhp8LhKGRg!GreLoMHHsVkwIcvpco7_fIFJAgJBpHYRw2ODkVLiLLTFT834Owq6V8GhroEyQXznYG2Uf5pfWOoZUaRrPOo4jWrj$>:

> @@ -35,7 +35,6 @@ EOF
     echo "CC                 Path to C compiler       (default is ${CC})"
     echo "CXX                Path to C++ compiler     (default is ${CXX})"
     echo "FC                 Path to Fortran compiler (default is ${FC})"
-    echo "BML_ROCSPARSE      Build with rocSPARSE     (default is ${BML_ROCSPARSE})"


Why do you remove this line? The variable BML_ROCSPARSE is still being used.

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/lanl/qmd-progress/pull/244*pullrequestreview-1286337475__;Iw!!Bt8fGhp8LhKGRg!GreLoMHHsVkwIcvpco7_fIFJAgJBpHYRw2ODkVLiLLTFT834Owq6V8GhroEyQXznYG2Uf5pfWOoZUaRrPHDSUBDd$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AA67VEP2UJZPN5DIKE34WFLWWG735ANCNFSM6AAAAAAUSXV35U__;!!Bt8fGhp8LhKGRg!GreLoMHHsVkwIcvpco7_fIFJAgJBpHYRw2ODkVLiLLTFT834Owq6V8GhroEyQXznYG2Uf5pfWOoZUaRrPPSUZIXC$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

jeanlucf22 reviewed (changes requested):
What do you mean? Are we dropping off support for rocsparse?

mewall commented:
The BML package now handles the rocSPARSE dependency. Please see the note on the commit with reference to the BML PR#...

________________________________
From: Jean-Luc Fattebert ***@***.***>
Sent: Tuesday, February 7, 2023 8:19:04 AM
To: lanl/qmd-progress
Cc: Wall, Michael E; Author
Subject: [EXTERNAL] Re: [lanl/qmd-progress] Debug magma on crusher (PR #244)


@jeanlucf22 requested changes on this pull request.

What do you mean? Are we dropping off support for rocsparse?

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/lanl/qmd-progress/pull/244*pullrequestreview-1287319071__;Iw!!Bt8fGhp8LhKGRg!Cgurs-KnpHTWOaT5-D7YvWAtXwu34CRZa2eS8tJnA7lf5Z1URilttfFzwve7xTmx2d-Da1tzhKaxiYZ1n5aH_Ky2$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AA67VEJVLBBZFEDGXZXOMITWWJRWRANCNFSM6AAAAAAUSXV35U__;!!Bt8fGhp8LhKGRg!Cgurs-KnpHTWOaT5-D7YvWAtXwu34CRZa2eS8tJnA7lf5Z1URilttfFzwve7xTmx2d-Da1tzhKaxiYZ1nz0yrX6s$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

mewall commented:
See


https://github.com/lanl/bml/blob/master/src/BMLConfig.cmakein

________________________________
From: Wall, Michael E
Sent: Tuesday, February 7, 2023 8:24:57 AM
To: lanl/qmd-progress; lanl/qmd-progress
Cc: Author
Subject: Re: [EXTERNAL] Re: [lanl/qmd-progress] Debug magma on crusher (PR #244)


The BML package now handles the rocSPARSE dependency. Please see the note on the commit with reference to the BML PR#...

________________________________
From: Jean-Luc Fattebert ***@***.***>
Sent: Tuesday, February 7, 2023 8:19:04 AM
To: lanl/qmd-progress
Cc: Wall, Michael E; Author
Subject: [EXTERNAL] Re: [lanl/qmd-progress] Debug magma on crusher (PR #244)


@jeanlucf22 requested changes on this pull request.

What do you mean? Are we dropping off support for rocsparse?

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/lanl/qmd-progress/pull/244*pullrequestreview-1287319071__;Iw!!Bt8fGhp8LhKGRg!Cgurs-KnpHTWOaT5-D7YvWAtXwu34CRZa2eS8tJnA7lf5Z1URilttfFzwve7xTmx2d-Da1tzhKaxiYZ1n5aH_Ky2$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AA67VEJVLBBZFEDGXZXOMITWWJRWRANCNFSM6AAAAAAUSXV35U__;!!Bt8fGhp8LhKGRg!Cgurs-KnpHTWOaT5-D7YvWAtXwu34CRZa2eS8tJnA7lf5Z1URilttfFzwve7xTmx2d-Da1tzhKaxiYZ1nz0yrX6s$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

jeanlucf22 commented:
@mewall sorry, I thought I was looking at BML. I agree, rocsparse should not be anywehere in the build scripts here

nicolasbock reviewed (changes requested):
Thanks for the clarification!

Can we remove

https://github.com/lanl/qmd-progress/blob/48c9a7c210c4a4e3028453118bf95aff166e8f1f/scripts/build_crusher_rocsparse_cce.sh#L16

also then?

mewall commented:
Done

________________________________
From: Nicolas Bock ***@***.***>
Sent: Tuesday, February 7, 2023 9:56:44 AM
To: lanl/qmd-progress
Cc: Wall, Michael E; Mention
Subject: [EXTERNAL] Re: [lanl/qmd-progress] Debug magma on crusher (PR #244)


@nicolasbock requested changes on this pull request.

Thanks for the clarification!

Can we remove

https://github.com/lanl/qmd-progress/blob/48c9a7c210c4a4e3028453118bf95aff166e8f1f/scripts/build_crusher_rocsparse_cce.sh#L16<https://urldefense.com/v3/__https://github.com/lanl/qmd-progress/blob/48c9a7c210c4a4e3028453118bf95aff166e8f1f/scripts/build_crusher_rocsparse_cce.sh*L16__;Iw!!Bt8fGhp8LhKGRg!Fl62TMZF5cxdIe2NvuQPjXhVMKr5P0GpGfL0vOny-VWqv3oymZogcRusMjZjWiD7fQwb3BTSiOCY2Re27erd8wYr$>

also then?

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https://github.com/lanl/qmd-progress/pull/244*pullrequestreview-1287526066__;Iw!!Bt8fGhp8LhKGRg!Fl62TMZF5cxdIe2NvuQPjXhVMKr5P0GpGfL0vOny-VWqv3oymZogcRusMjZjWiD7fQwb3BTSiOCY2Re27bv-C1uk$>, or unsubscribe<https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/AA67VELARYJY53XYDGFI2LLWWJ5EZANCNFSM6AAAAAAUSXV35U__;!!Bt8fGhp8LhKGRg!Fl62TMZF5cxdIe2NvuQPjXhVMKr5P0GpGfL0vOny-VWqv3oymZogcRusMjZjWiD7fQwb3BTSiOCY2Re27QhyR3_b$>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

nicolasbock reviewed (approved):
Thanks!
